Maintenace window 2.0 short (#20901)
pingcap/docs: tidb-cloud/configure-maintenance-window.md

Regular maintenance ensures that essential updates are performed to safeguard TiDB Cloud from security threats, performance issues, and unreliability. Therefore, the maintenance window is enabled by default and cannot be disabled.

> **Note:**
>

> > For the default project automatically created when you first sign up TiDB Cloud, the default start time of the window is 03:00 AM every Wednesday, based on the time zone of your TiDB Cloud organization. For any new projects that you initiate, you have the option to customize the start time of the window during the project setup process.

Reviewer: ```suggestion
> - For the default project automatically created when you first sign up for TiDB Cloud, the maintenance window starts at 03:00 AM every Wednesday (based on the time zone of your TiDB Cloud organization).
> - For new projects that you create, you can set a custom start time for the maintenance window during project setup.
```

     - If there is no maintenance data, it means no maintenance task is scheduled recently.

4. (Optional) Click **Maintenance Window Setting** to modify the start time of the maintenance window. Note that the maintenance will be performed at the specified start time only if there is a maintenance window planned for that week.

> 5. To reschedule the start time of a scheduled maintenance task, click **...** and **Reschedule** in the **Action** column and change it to your preferred time before the deadline.

Reviewer: ```suggestion
5. To reschedule a specific maintenance task, click **...** > **Reschedule** in the **Action** column, and choose a new time before the deadline.
```
